Readability.js

```diff
@@ -1233,20 +1224,39 @@ Readability.prototype = {
       }
     });
 
-    if ("description" in values) {
-      metadata.excerpt = values["description"];
+    // get author
+    if ("author" in values) {
+      metadata.excerpt = values["author"];
+    } else if ("dc:author" in values) {
+      // Use dublin core author.
+      metadata.excerpt = values["dc:author"];
+    } else if ("article:author" in values) {
```

Comment:
I'm probably missing something, but it doesn't look to me like a meta tag with `article:author` would pass the regular expression you've used above - I assume the `author` in the first group in the regex should be `article` instead. In fact, if I'm not missing something:

```js
 /^\s*(?:author|dc|og|twitter)\s*:\s*(author|description|title)\s*$/gi;
```
looks to me like it won't match plain `"author"` either, because you've removed the question mark that used to be after the first group.

Reply:
Ah, also, as pointed out by someone else in #480, can you remove the `g` flag from the new regex? It shouldn't be necessary and it causes bugs (like that one).